xml/deploy_cephadm.xml

```diff
@@ -701,131 +701,287 @@ o- time_server ................................................ [enabled]
+<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/password set <replaceable>REG_PASSWORD</replaceable></screen>
+      </step>
+     </procedure>
+     <tip>
+      <title>Registry cache</title>
       <para>
-       Run <command>ceph-salt apply</command> to update the &salt; pillar on
-       all minions.
+       To avoid re-syncing the local registry when new updated containers
+       appear, you can configure a <emphasis>registry cache</emphasis>.
       </para>
-     </step>
-    </procedure>
-    <tip>
-     <title>Registry cache</title>
+     </tip>
+    </sect4>
+    <sect4 xml:id="deploy-cephadm-configure-imagepath">
+     <title>Configuring the path to container images</title>
+     <tip>
+      <title>Configuring HTTPS proxy</title>
+      <para>
+       If you need to use a proxy to communicate with the container registry
+       server, perform the following configuration steps on all cluster nodes:
+      </para>
+      <procedure>
+       <step>
+        <para>
+         Copy the configuration file for containers:
+        </para>
+<screen>&prompt.sudo;cp /usr/share/containers/containers.conf /etc/containers/containers.conf</screen>
+       </step>
+       <step>
+        <para>
+         Edit the newly-copied file and add the <option>http_proxy</option>
+         setting to its <literal>[engine]</literal> section, for example:
+        </para>
+<screen>&prompt.user;cat /etc/containers/containers.conf
+ [engine]
+ http_proxy=proxy.example.com
+ [...]
+ </screen>
+       </step>
+      </procedure>
+     </tip>
      <para>
-      To avoid re-syncing the local registry when new updated containers
-      appear, you can configure a <emphasis>registry cache</emphasis>.
-<!-- 2021-02-09 tbazant: still looking for better solution
-      Find
-      more information about containers and air-gapped scenarios in
-      <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/#_deployment_in_an_air_gapped_environment"/>.
-      -->
+      &cephadm; needs to know a valid URI path to container images. Verify the
+      default setting by executing
      </para>
-    </tip>
-    <para>
-     Cloud Native Application Development and Delivery methods require a
-     registry and a CI/CD (Continuous Integration/Delivery) instance for the
-     development and production of container images. You can use a private
-     registry in this instance.
-<!-- 2021-02-09 tbazant: still looking for better solution
-     For more information, see
-     <link
-       xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/"/>.
-     -->
-    </para>
+<screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_image_path ls</screen>
+     <para>
+      If you do not need an alternative path or to create a local container
+      registry, you can leave the default or set it when the setting has no
+      default path:
```

Comment:
I do not understand this paragraph. I think it is very confusing. It appears to me to be conflating 2 situations & 2 or 3 answers.

Situation 1: you don't need an alternative path
Sit 2: you don't need a local registry

Answer 1: leave the default path
Ans 2: set it when the setting has no default path (which I also don't understand)

At the very least, **if** these situations are **not** interrelated, I think it would be cleaner split into (at least) 2 separate sentences:

«
The default setting is no path. **[If this is correct!]**
If you do not need an alternative path, or you do not have a local registry, you do not need to change this.
If you have a local registry, enter the path to it as follows:
»

Reply:
It can't be left blank, it needs to be set to _something_. If you're not using a local registry, you would ordinarily set it to point to the image on the SUSE registry, i.e. `registry.suse.com/ses/7/ceph/ceph`. If you're using a local registry, you'd set it to the path to the ceph image on the local registry.